reader_test.go

```diff
@@ -1313,3 +1313,45 @@ func getOffsets(t *testing.T, config ReaderConfig) offsetFetchResponseV1 {
 
 	return offsets
 }
+
+const (
+	connTO     = 1 * time.Second
+	connTestTO = 2 * connTO
+)
+
+func TestErrorCannotConnect(t *testing.T) {
+	r := NewReader(ReaderConfig{
+		// FIXME
```

Comment:
What needs to be fixed here?  We run tests using this docker-compose setup: https://github.com/segmentio/kafka-go/blob/master/docker-compose.yml.  The way I read it, I would expect this test to pass b/c port 9093 requires SASL authentication and would reply with an error.

One other question: if there's no consumer group ID set, are we exercising the logic being adding?

Reply:
Nothing, really. I think I just forgot to remove the FIXME.